src/Microsoft.DotNet.Cli.Utils/CommandResolution/PackagedCommandSpecFactoryWithCliRuntime.cs

```diff
@@ -21,9 +21,41 @@ private static void AddAditionalParameters(string commandPath, IList<string> arg
         {
             if(PrefersCliRuntime(commandPath))
             {
-                arguments.Add("--fx-version");
-                arguments.Add(new Muxer().SharedFxVersion);
+                var runtimeConfigFile = Path.ChangeExtension(commandPath, FileNameSuffixes.RuntimeConfigJson);
```

Comment:
Is it allowed for tools to not have a runtimeconfig file?  My thought at the time was that this was required, although thinking about it again it seems like it could be OK not to include this file if you have prefercliruntime.

If it's OK for a tool not to have a runtimeconfig file then we'd need another way to check what version of .NET Core it targets, in order to prevent rolling forward.  We might be able to look at the path in the NuGet package, or look at the TargetFrameworkAttribute on the assembly.